README.md

```diff
@@ -1 +1,24 @@
-> This a test for Creating a Rails-React App
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```

Comment:
[Optional] Kindly give a more descriptive README about your project, This gives insight about your project and professionalism 👍